services/wallet/api.go

```diff
@@ -712,6 +716,15 @@ func (api *API) SendTransactionWithSignature(ctx context.Context, chainID uint64
 	return api.s.transactionManager.SendTransactionWithSignature(chainID, params, sig)
 }
 
+// @deprecated `CreateMultiTransaction`
```

Comment:
I just saw this today in the `wallet/api.go`. Not a big deal, just sharing info
It should be `// Deprecated: ...` as per https://go.dev/wiki/Deprecated

Reply:
Updated for endpoints marked as deprecated in this PR.